docs/resources/dli_permission.md

```diff
@@ -0,0 +1,105 @@
+---
+subcategory: "Data Lake Insight (DLI)"
+---
+
+# huaweicloud_dli_permission
+
+Manages the usage permissions of those resources: `huaweicloud_dli_queue`, `huaweicloud_dli_database`,
+ `huaweicloud_dli_table`, `huaweicloud_dli_package`, `huaweicloud_dli_flinksql_job`, `huaweicloud_dli_flinkjar_job`
+  within HuaweiCloud DLI.
+
+## Example Usage
+
+### Grant a permission of queue
+
+```hcl
+variable "user_name" {}
+variable "queue_name" {}
+
+resource "huaweicloud_dli_permission" "test" {
+  user_name  = var.user_name
+  object     = "queues.${var.queue_name}"
+  privileges = ["SUBMIT_JOB","DROP_QUEUE"]
+}
+```
+
+### Grant a permission of database
+
+```hcl
+variable "user_name" {}
+variable "database_name" {}
+
+resource "huaweicloud_dli_permission" "test" {
+  user_name  = var.user_name
+  object     = "databases.${var.database_name}"
+  privileges = ["SELECT"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the DLI permission resource. If omitted, the
+  provider-level region will be used. Changing this parameter will create a new resource.
+
+* `user_name` - (Required, String, ForceNew) Specifies name of the user who is granted with usage permission.
+ Changing this parameter will create a new resource.
+
+* `object` - (Required, String, ForceNew) Specifies which object's data usage permissions will be shared.
+  Its naming format is as follows:
+  + **queues.`queues_name`**: the usage permissions of queue.
+  + **databases.`database_name`**: the usage permissions of data in the database.
+  + **databases.`database_name`.tables.`table_name`**: the usage permissions of data in the table.
+  + **databases.`database_name`.tables.`table_name`.columns.`column_name`**: the usage permissions of data in the column.
+  + **jobs.flink.`flink_job_id`**: the usage permissions of data in the flink job.
+  + **groups.`package_group_name`**: the usage permissions of data in the package group.
+  + **resources.`package_name`**: the usage permissions of data in the package.
+
+  Changing this parameter will create a new resource.
+
+* `privileges` - (Required, List) Specifies the usage permissions of data.
+  + Permissions on Queue, Database and Table,
+   please see [Permissions Management](https://support.huaweicloud.com/intl/en-us/productdesc-dli/dli_07_0006.html)
```

Comment:
Maybe `For more details of the permissions on queue, database and table, please see permissions management [document](......).` is better.